src/membarrier.h

```diff
@@ -48,6 +48,14 @@
 # define RMB asm volatile (".arch armv8.1-a \n\t dsb sy\n" : : : "memory")
```

Comment:
We have a later commit with the commit msg:
> Use membarrier

It seems like that's related to this commit.  Either we want to do both commits or neither of them.
Please add the later commit, `Use membarrier` to this commit.

Also, this commit msg is 52 chars (too long).  Please change to something like:
> riscv: Defining inline functions RMB, WMB and IMB